.github/workflows/reusable_build_web_demo.yml

```diff
@@ -0,0 +1,84 @@
+name: Reusable Build Web Demo
+
+on:
+  workflow_call:
+    inputs:
+      CONCURRENCY:
+        required: true
+        type: string
+      WHEEL_ARTIFACT_NAME:
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ inputs.CONCURRENCY }}-build-web-demo
+  cancel-in-progress: true
+
+env:
+  PYTHON_VERSION: "3.8"
+  # web_sys_unstable_apis is required to enable the web_sys clipboard API which egui_web uses
+  # https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.Clipboard.html
+  # https://rustwasm.github.io/docs/wasm-bindgen/web-sys/unstable-apis.html
+  RUSTFLAGS: --cfg=web_sys_unstable_apis --deny warnings
+
+  # See https://github.com/ericseppanen/cargo-cranky/issues/8
+  RUSTDOCFLAGS: --deny warnings --deny rustdoc::missing_crate_level_docs
+
+jobs:
+  rs-build-web-viewer:
+    name: Build Web Demo
+    permissions:
+      contents: "read"
+      id-token: "write"
+
+    runs-on: ubuntu-latest-16-cores
+
+    container:
+      image: rerunio/ci_docker:0.6
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download Web Viewer
+        uses: actions/download-artifact@v3
+        with:
+          name: web_viewer
+          path: web_viewer
+
+      - name: Download Wheel
+        uses: actions/download-artifact@v3
+        with:
+          name: ${{ inputs.WHEEL_ARTIFACT_NAME }}
+          path: wheel
+
+      - name: Install dependencies for examples/python
+        run: |
+          pip install -r scripts/requirements-web-demo.txt
+          pip install Jinja2
+
+      - name: Install built wheel
+        shell: bash
+        run: |
+          pip uninstall rerun-sdk -y
+          pip install deprecated numpy>=1.23 pyarrow==10.0.1 pytest==7.1.2
+          pip install rerun-sdk --no-index --find-links wheel
+
+      - name: Verify built wheel version
+        shell: bash
+        run: |
+          python3 -m rerun --version
+          which rerun
+          rerun --version
+
+      - name: Build web demo
+        env:
+          COMMIT_HASH: ${{ github.env.GITHUB_SHA }}
```

Comment:
Don't you need to do the whole commit override dance here? Asking just in case, I'm not too familiar with the matter...

Reply:
I'm not sure, either. Maybe @jleibs can comment on this?

Reply:
I believe so.  Without the commit-override-dance then when running this job **on pull request*, GITHUB_SHA is going to sometimes point to an ephemeral merge-commit, which I think means your code-link from the examples will be broken.

Not necessarily the end of the world for this use-case since the adhoc / on-push-main versions which are the only ones I expect someone in the public to see will still do the right thing, but probably worth adding in the support.

Reply:
Added this in the form of a `SOURCE_LINK_COMMIT_OVERRIDE` input to the `build_web_demo` workflow.